src/main/java/forestry/core/config/Config.java

```diff
@@ -97,6 +100,37 @@ public class Config {
     public static GuiMailboxInfo.XPosition mailAlertXPosition = GuiMailboxInfo.XPosition.LEFT;
     public static GuiMailboxInfo.YPosition mailAlertYPosition = GuiMailboxInfo.YPosition.TOP;
 
+    public static List<Item> mailBlacklist = new ArrayList<Item>();
```

Comment:
Why do you use `List` and not `Set`?
I think that with such large blacklist `contains` checks with `ArrayList` will be slow.

Reply:
Thanks for noticing that. 
You make a good point and I should go and change that